Description: usr-local removal
 remove usr local, install to usr like dh_source expects
 .
 glclib (3.0.0-1) UNRELEASED; urgency=medium
 .
   * Initial release.
Author: Alon Levy <alon.levy@bladeranger.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- glclib-2.3.0.orig/install.pri
+++ glclib-2.3.0/install.pri
@@ -4,8 +4,8 @@ GLC_LIB_NAME = GLC_lib-3.0
 unix {
     # Location of HEADERS and library
     GLC_LIB_DIR = $$(HOME)/$${GLC_LIB_NAME}
-    LIB_DIR = /usr/local/lib
-    INCLUDE_DIR = /usr/local/include/$${GLC_LIB_NAME}
+    LIB_DIR = /usr/lib
+    INCLUDE_DIR = /usr/include/$${GLC_LIB_NAME}
     # Adds a -P to preserve link
         QMAKE_COPY_FILE = $${QMAKE_COPY_FILE} -P
 }
